container_pipeline/lib/command.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+import subprocess
+
+
+def run(cmd):
+    """
+    Runs a shell command.
+
+    :param cmd: Command to run
+    :type cmd: str
+
+    :return: Command output
+    :rtype: str
+    :raises: subprocess.CalledProcessError
+    """
+    return subprocess.check_output(cmd, shell=True)
```

Comment:
We use pipes in a number of commands, so, I believe that we'll need ``shell`` support. Also, it makes it easy to call the method. That said, I don't disagree with the potential misuse of it. However, our system does not have much user interaction from the outside world. The point of entry to our system is jenkins and the jobs are written by us. So, I don't see much room for exploit. The only variables that we have is the container-index params, which can be sanitized during linting the index.

However, this calls for a discussion.